src/shared/components/login/socialLogin.js

```diff
@@ -11,7 +17,64 @@ let zipCode;
 let pass;
 
 class SocialLogin extends React.Component {
-  static login() {
+  state = {
+    zip: '',
+    password: ''
+  }
+  run(f, l, e) {
```

Comment:
Can we use more meaningful argument names here, instead of single letters or abbreviations.